client/api/two/webhooks.py

```diff
@@ -0,0 +1,48 @@
+from client.resource import Resource
+
+
+class Webhooks(Resource):
+
+    def all(self, page=None, limit=20):
+        params = {
+            'page': page,
+            'limit': limit
+        }
+        return self.request_get('webhooks', params)
+
+    def create(self, event, target, conditions, enabled=None):
+        data = {
+            'event': event,
+            'target': target,
+            'conditions': conditions
+        }
+        if enabled is not None:
```

Comment:
you can do `if isisntance(enabled, bool)`, I suspect enabled can either be `True`/`False`? Also would API return us an error if we were to pass enabled set to None to it?

Reply:
>Also would API return us an error if we were to pass enabled set to None to it?

Ah, looks like we can safely pass `None` as the value for this to the REST API.